config/rubocop-2.4.yml

```diff
@@ -103,6 +103,10 @@ Layout/EmptyLines:
   Description: Don't use several empty lines in a row.
   Enabled: true
 
+Layout/EmptyLineAfterGuardClause:
```

Comment:
This should be above `EmptyLineBetweenDefs` since it's alphabetical

Reply:
Definitely, I'll make sure to double check the other cops as well.